catroid/src/main/res/layout/dialog_new_project.xml

```diff
@@ -53,33 +53,45 @@
             android:id="@+id/radio_group"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:checkedButton="@+id/project_empty_radio_button">
+            android:checkedButton="@+id/portrait"
```

Comment:
Please append _radio_button as it was before. In order to make it more readable to distinguish the different types of views.